dev-libs/efl/efl-9999.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Also, I think it'd be nicer if you started with doing all the necessary changes in -9999 (as one commit), then making the bump as a separate commit. You may delay adding the actual bump until we're past reviewing changes for the ebuild.

Reply:
9999 need major adjustment for the switch to meson. This will be done after we are done with this request, for now keeping 9999 and rev in sync helps